functional-test/src/test/groovy/org/rundeck/tests/functional/selenium/webhook/WebhooksSpec.groovy

```diff
@@ -36,4 +41,181 @@ class WebhooksSpec extends SeleniumBase {
         deleteProject(projectName)
     }
 
+    /**
+     * Create webhook in a project
+     */
+    void "create project with webhook"() {
+        given:
+        String projectName = 'CreateWebhookProject'
+        String webhookName = 'CreateWebhook'
+        setupProject(projectName)
+        WebhooksPage webhookPage = page WebhooksPage
+        LoginPage loginPage = go LoginPage
+        HomePage homePage = page HomePage
+
+        when:
+        loginPage.login(TEST_USER, TEST_PASS)
+        homePage.validatePage()
+        webhookPage.loadPageForProject(projectName)
+        createSimpleWebhook(webhookPage, webhookName)
+        webhookPage.waitForNumberOfElementsToBe(webhookPage.alertInfoBy, 0)
+        removeWebhook(webhookPage, webhookName)
```

Comment:
here it should validate that the webhook does not exist

Reply:
Done...✅